internal/eval/eval_test.go

```diff
@@ -3964,137 +3964,181 @@ func testEvalGETRANGE(t *testing.T, store *dstore.Store) {
+		{
+			name:  "GETRANGE against integer value: 3, 5000",
 			setup:  setupForIntegerValue,
 			input:  []string{"INTEGER_KEY", "3", "5000"},
-			output: clientio.Encode("4", false),
+			migratedOutput: EvalResponse{Result: "4", Error: nil},
 		},
-
-		"GETRANGE against integer value: -5000, 10000": {
-			setup:  setupForIntegerValue,
-			input:  []string{"INTEGER_KEY", "-5000", "10000"},
-			output: clientio.Encode("1234", false),
+		{
+			name: "GETRANGE against integer value: -5000, 10000",
+			setup: setupForIntegerValue,
+			input: []string{"INTEGER_KEY", "-5000", "10000"},
+			migratedOutput: EvalResponse{Result: "1234", Error: nil},
 		},
-		"GETRANGE against integer value: 0, -100": {
-			setup:  setupForIntegerValue,
-			input:  []string{"INTEGER_KEY", "0", "-100"},
-			output: clientio.Encode("", false),
+		{
+			name: "GETRANGE against integer value: 0, -100",
+			setup: setupForIntegerValue,
+			input: []string{"INTEGER_KEY", "0", "-100"},
+			migratedOutput: EvalResponse{Result: "", Error: nil},
 		},
-		"GETRANGE against integer value: 1, -100": {
-			setup:  setupForIntegerValue,
-			input:  []string{"INTEGER_KEY", "1", "-100"},
-			output: clientio.Encode("", false),
+		{
+			name: "GETRANGE against integer value: 1, -100",
+			setup: setupForIntegerValue,
+			input: []string{"INTEGER_KEY", "1", "-100"},
+			migratedOutput: EvalResponse{Result: "", Error: nil},
 		},
-		"GETRANGE against integer value: -1, -100": {
-			setup:  setupForIntegerValue,
-			input:  []string{"INTEGER_KEY", "-1", "-100"},
-			output: clientio.Encode("", false),
+		{
+			name: "GETRANGE against integer value: -1, -100",
+			setup: setupForIntegerValue,
+			input: []string{"INTEGER_KEY", "-1", "-100"},
+			migratedOutput: EvalResponse{Result: "", Error: nil},
 		},
-		"GETRANGE against integer value: -100, -99": {
-			setup:  setupForIntegerValue,
-			input:  []string{"INTEGER_KEY", "-100", "-99"},
-			output: clientio.Encode("", false),
+		{
+			name: "GETRANGE against integer value: -100, -99",
+			setup: setupForIntegerValue,
+			input: []string{"INTEGER_KEY", "-100", "-99"},
+			migratedOutput: EvalResponse{Result: "", Error: nil},
 		},
-		"GETRANGE against integer value: -100, -100": {
-			setup:  setupForIntegerValue,
-			input:  []string{"INTEGER_KEY", "-100", "-100"},
-			output: clientio.Encode("", false),
+		{
+			name: "GETRANGE against integer value: -100, -100",
+			setup: setupForIntegerValue,
+			input: []string{"INTEGER_KEY", "-100", "-100"},
+			migratedOutput: EvalResponse{Result: "", Error: nil},
 		},
-		"GETRANGE against integer value: -100, -101": {
-			setup:  setupForIntegerValue,
-			input:  []string{"INTEGER_KEY", "-100", "-101"},
-			output: clientio.Encode("", false),
+		{
+			name: "GETRANGE against integer value: -100, -101",
+			setup: setupForIntegerValue,
+			input: []string{"INTEGER_KEY", "-100", "-101"},
+			migratedOutput: EvalResponse{Result: "", Error: nil},
 		},
 	}
-	runEvalTests(t, tests, evalGETRANGE, store)
+
```

Comment:
We have a function called `runMigratedEvalTests` that does this same thing and can be reused here.